default keymap fix for questionmark

biggestj opened the pull request:
Added /? to the default position for a qwerty keyboard. Moved |\ to the left ctrl with the same behaviour as before. This pertains to the wireless and the wired variants.

## Types of Changes

- [X] Bugfix
- [X] Keymap/layout/userspace (addition or update)

## Checklist

- [X] My code follows the code style of this project.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

biggestj commented:
@mattdibi I assume `KC_BLSLASH` doesn't have the | (pipe) character under it's shift?

mattdibi commented:
@biggestj  [It does](https://beta.docs.qmk.fm/features/keycodes_basic).

biggestj commented:
Okay, also updating `KC_SLSH` to `KC_SLASH` for consistency

mattdibi commented:
@biggestj Yup, use the ones provided by QMK over the one created in the keymap. Use the shorthand versions when possible to avoid disrupting the keymap looks.

biggestj commented:
> @biggestj Yup, use the ones provided by QMK over the one created in the keymap. Use the shorthand versions when possible to avoid disrupting the keymap looks.

The shorthand code for backslash is still `KC_BSLS` in that document. See: https://beta.docs.qmk.fm/features/keycodes_basic#punctuation

mattdibi commented:
@biggestj I'm sorry, I need to clarify: in the [keymap](https://github.com/qmk/qmk_firmware/blob/6a79b05cf02d5a6f07b17474344e272b44c77988/keyboards/redox_w/keymaps/default/keymap.c#L36) I used all the shorthand versions because I needed a consistent look to make it easier to read. When I couldn't (example: `LT(_ADJUST, KC_PGUP)`) I created [my own custom shortands](https://github.com/qmk/qmk_firmware/blob/6a79b05cf02d5a6f07b17474344e272b44c77988/keyboards/redox_w/keymaps/default/keymap.c#L22), and in the definition of those shorthands I would use the longer version of the keycode for clarity. Does this make sense?

fauxpark commented:
@biggestj if you could also remove the `#define KC_BKSL KC_BSLASH` lines from both keymaps (as they are redundant and now unused), this seems good to merge. And for bonus points, the `extern keymap_config_t keymap_config;`s are not referenced anywhere either.

biggestj commented:
Sure I've got some time tomorrow evening. Will work on it then.

biggestj commented:
@mattdibi I saw what you meant now. Completely read over that definition. Adjusted it now. Also changed the keypad to shorthand and added the rest of the keypad keys. (to the sides might be a good idea to change that to the F-row, but that might be too confusing without changing the entire rows behaviour)

@fauxpark Made those changes, should be good to go now.

biggestj commented:
Should be fixed now @noroadsleft
